src/test/java/com/thealgorithms/datastructures/graphs/WelshPowellAlgorithmTest.java

```diff
@@ -0,0 +1,63 @@
+package com.thealgorithms.datastructures.graphs;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.thealgorithms.datastructures.graphs.WelshPowell.Graph;
+import java.util.Arrays;
+import org.junit.jupiter.api.Test;
+
+class WelshPowellAlgorithmTest {
```

Comment:
```suggestion
class WelshPowellTest {
```

I would be consistent. If there is a `SomeClass`, `SomeClassTest` should test it.